contracts/ServiceManagerToken.sol

```diff
@@ -32,14 +32,22 @@ contract ServiceManagerToken is GenericManager {
     // Operator whitelist address
     address public operatorWhitelist;
 
+    /// @dev ServiceRegistryTokenUtility constructor.
+    /// @param _serviceRegistry Service Registry contract address.
+    /// @param _serviceRegistryTokenUtility Service Registry Token Utility contract address.
     constructor(address _serviceRegistry, address _serviceRegistryTokenUtility) {
```

Comment:
Ok, here we will also add the operator whitelist address and the corresponding non-zero check